aries-site/src/data/structure.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable max-len */
+import {
+  IconComponents,
+  IconDesign,
+  IconDevelop,
+  IconFoundation,
+  IconGuidelines,
+  IconResources,
+  IconBrand,
+  IconColor,
+  IconIcons,
+  IconLayout,
+  IconType,
+  IconToken,
+} from '../components/icons';
+
+export const structure = [
```

Comment:
Now that this is flattened, `pages` might be more precise that `structure`. Not a big deal either way.

Reply:
I like pages. I'll make a switch.